app/main/auth.py

```diff
@@ -0,0 +1,30 @@
+from functools import wraps
+
+from flask import abort
+from flask_login import current_user
+
+
+def role_required(*roles):
+    """Ensure that logged in user has one of the required roles.
+
+    Return 403 if the user doesn't have a required role.
+
+    Should be applied before the `@login_required` decorator:
+
+        @login_required
+        @role_required('admin', 'admin-ccs')
```

Comment:
I'm not sure about specifying 'CCS' in the role name - it might be nicer to have the role describe what permissions the role gives, rather than who the users with that role are, e.g. 'admin-readonly' or 'admin-view' (I guess we may have non-ccs people with this role in future?)